src/js/timeline.js

```diff
@@ -1,80 +1,69 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+class Refresh extends React.Component {
+  refreshTimeline() {
+    getTL();
+  }
+  render() {
+    return React.createElement('button', {type: 'button', onClick: this.refreshTimeline}, "Refresh Timeline");
```

Comment:
add newline characters to do the button properties
i.e.
...React.createElement('button', {
  type:button,
  onlick...
}

Reply:
nice job getting the button. Can you have a better component name?